api/Node.json

```diff
@@ -1603,7 +1616,7 @@
               "notes": "Only supported on <code>Element</code>."
             },
             "opera": {
-              "version_added": "7"
+              "version_added": "15"
```

Comment:
This can't be right, Presto definitely had `parentElement`. It might have been only on `Element`, perhaps, like IE?

Reply:
Ack, this update slipped by -- ah well, I'll add a note and revert the version change!